vertica/datadog_checks/vertica/vertica.py

```diff
@@ -92,6 +92,37 @@ def __init__(self, name, init_config, instances):
         # Cache database results for re-use among disparate functions
         self._view = defaultdict(list)
 
+        self._metric_groups = {
+            "licenses": self.query_licenses,
+            "license_audits": self.query_license_audits,
+            "system": self.query_system,
+            "nodes": self.query_nodes,
+            "projections": self.query_projections,
+            "projection_storage": self.query_projection_storage,
+            "storage_containers": self.query_storage_containers,
+            "host_resources": self.query_host_resources,
+            "query_metrics": self.query_query_metrics,
+            "resource_pool_status": self.query_resource_pool_status,
+            "disk_storage": self.query_disk_storage,
+            "resource_usage": self.query_resource_usage,
+            "version": self.query_version,
+            "custom": self.query_custom,
+        }
+
+        include_groups = self.instance.get("metric_groups", [])
```

Comment:
Since this is a required field we should check in `__init__` that it is correctly set, throw a ConfigurationError if not